tests/unittests/tests-fmt/tests-fmt.c

```diff
@@ -184,7 +184,7 @@ static void test_fmt_s16_dec(void)
 
 static void test_fmt_s16_dfp(void)
 {
-    char out[8] = "--------";
+    char out[13] = "-------------";
```

Comment:
The string is too long for the buffer, the terminating null byte does not fit.

Reply:
mhm, could be `char out[] = "-------------";` anyway, to omit such errors?

Reply:
btw. this applies to **all** buffers in this test.

Reply:
maybe not all, but many